client/src/components/History/Multiple/MultipleViewItem.vue

```diff
@@ -30,6 +36,18 @@ const sameToCurrent = computed(() => {
     return currentHistoryId.value === props.source.id;
 });
 
+const switchToText = computed(() => {
+    if (sameToCurrent.value) {
+        return "Current History";
+    } else if (history.value?.purged) {
```

Comment:
You can also switch to a "deleted non-purged" history If I recall correctly.

Reply:
Oh I see, it was throwing an error but let me investigate and allow that as well

Reply:
Done, now this part is left unchanged and we can switch to purged or archived histories.